spec/system/admins/reward/create_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+describe 'Reward create action', type: :system do
+  before do
+    sign_in admin
+  end
+
+  let(:admin) { create(:admin) }
+
+  context 'when admin is on index page' do
+    it 'New Reward link' do
+      visit admins_rewards_path
+      click_on 'New Reward'
+      expect(page).to have_current_path new_admins_reward_path
+    end
+  end
+
+  context 'when admin is on new reward page' do
+    it 'displays reward form' do
```

Comment:
You do not have to check, if the fields are displayed here - you are filling them in the next test, so the next step will fail, if the fields won't be there :)